medicines/web/src/pages/substance-index/index.tsx

```diff
@@ -49,6 +51,25 @@ const App: NextPage = () => {
     >
       <SearchWrapper initialSearchValue="">
         <DrugIndex title={`${substanceIndex || '...'}`} items={results} />
+        <JsonLd<ItemList>
+          item={{
+            '@context': 'https://schema.org',
+            '@type': 'ItemList',
+            itemListElement: results.map((substance, index) => {
+              return {
+                '@type': 'ListItem',
+                position: index,
+                item: {
+                  '@type': 'Substance',
+                  name: substance.name,
+                  url:
+                    'https://products.mhra.gov.uk/substance?query=' +
+                    encodeURIComponent(substance.name),
+                },
+              };
+            }),
+          }}
+        />
```

Comment:
May be worth extracting this to a separate, self-contained component in the same file to make the contents of this component cleaner.

Reply:
Sounds like a good idea.